web/src/js/constants.js

```diff
@@ -126,3 +126,9 @@ export const SEARCH_SRC_OWN_PAGE = 'self'
 
 export const SEARCH_INTRO_QUERY_ENGLISH =
   'How many searches do people make every day?'
+
+export const STORAGE_YAHOO_SEARCH_DEMO = 'tab.yahoo.searchdemo'
+// local testing
+export const YAHOO_USER_ID = 'VXNlcjpjTDVLY0ZLSGQ5ZkVVNUM5VnN0ajNnNEpBYzcz'
```

Comment:
@kmjennison you'll want to test this locally by pulling down my branch then setting your user id here

Reply:
We can also make this a build env variable as well so we can change it based on env.

Reply:
Fine to leave as-is and test this live on prod.